### PR TITLE
Create makeSlackDark.bat for Windows 10 automatic install

### PR DESCRIPTION
**Pull request by Catscratch**

### CODE REVIEW

File: makeSlackDark.bat [hunk 1]
```diff
@@ -0,0 +1,3 @@
+echo off
+type static-css-injections.js >> %LOCALAPPDATA%\slack\app-3.1.1\resources\app.asar.unpacked\src\static\index.js
```
Comment (ariataylor96) [inline makeSlackDark.bat:2]: Doesn't the `app-3.1.1` part change whenever slack updates?

**Comment by Catscratch** [comment]
Yes

Vom Handy gesendet.



Am 24. Mai 2018 23:04:44 schrieb Mark Rawls <notifications@github.com>:

File: makeSlackDark.bat (hunk 1 above)
Comment (Catscratch) [reply makeSlackDark.bat:2]: Yep. But since the theme usually needs an update anyway, there should be no problem updating all install scripts.

Comment (d-fay) [reply makeSlackDark.bat:2]: Neat! Nice contrib

**Pull request merged**